docs/Installation-Windows.md

```diff
@@ -126,19 +126,28 @@ git clone https://github.com/Unity-Technologies/ml-agents.git
 If you don't want to use Git, you can always directly download all the files
 [here](https://github.com/Unity-Technologies/ml-agents/archive/master.zip).
 
+The `UnitySDK` subdirectory contains the Unity Assets to add to your projects.
```

Comment:
Why are we referring to the SDK as assets?

Reply:
I copied over the same sentences in the Installation doc to this Installation windows doc. Also they can be considered as Assets since they are something we can put in the Asset store. (And are compatible with the Asset usage, while the python package isn't).